tests/torch/requirements.txt

```diff
@@ -16,3 +16,4 @@ datasets~=2.14.0
 evaluate==0.3.0
 timm==0.9.2
 openvino-dev==2024.0
+pytest-xdist==3.5.0
```

Comment:
Don't forget to add "pytest-xdist" to the constraints file https://github.com/openvinotoolkit/nncf/pull/2581

Reply:
done